src/components/account-recovery/check-your-email-security-codes/send-email-otp-middleware.ts

```diff
@@ -39,6 +44,18 @@ export function sendEmailOtp(
     ) {
       return res.render("security-code-error/index-wait.njk");
     }
+    if (
```

Comment:
This won't only show if support2hrLockout() is true so counts as a change to live.  Is that correct?

Reply:
My intuition is that this code should be here, but I'll wrap it in a feature switch